models/notes.model.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const notesSchema = new Schema(
+	{
+		userId: { type: Schema.Types.ObjectId, ref: "User" },
+		video: { type: Schema.Types.ObjectId },
+		notes: String,
+	},
+	{ timestamps: true }
```

Comment:
notes are not being referenced in either video schema or in user schema, If referenced in video schema it can decrease a network call I guess, But its just a suggestion

Reply:
It is not referenced anywhere, so no request will be fired